testdata/ci/features/Job.feature

```diff
@@ -0,0 +1,37 @@
+#
+# 2018 Bull S.A.S. Atos Technologies - Bull, Rue Jean Jaures, B.P.68, 78340, Les Clayes-sous-Bois, France.
+#
+
+@Job
+Feature: Deploy a TestJobApp application using Alien4Cloud with a job mock
+
+  Background: Deploying required artifacts
+    Given I have uploaded the artifact named "org-ystia-samples-jobmock" to Alien
+    And I have built the artifact named "testJob" from templates named "testJob" to Alien
+    And I have uploaded the artifact named "testJob" to Alien
+
+  @CI
+  @hp
```

Comment:
HostsPool only?

Reply:
Yes but it can be extended for all infras

Reply:
Done